charts/opentelemetry-operator/templates/_helpers.tpl

```diff
@@ -24,9 +24,5 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 Allow the release namespace to be overridden for multi-namespace deployments in combined charts
 */}}
 {{- define "opentelemetry-operator.namespace" -}}
-  {{- if .Values.namespaceOverride -}}
-    {{- .Values.namespaceOverride -}}
-  {{- else -}}
-    {{- .Release.Namespace -}}
-  {{- end -}}
+  "opentelemetry-operator-system"
 {{- end -}}
```

Comment:
Should we still have this named template instead of using `opentelemetry-operator-system` string? If you want to keep it, let's update the comment above.

Reply:
I think we can keep it since we want to hard-code it. I will update the comment.